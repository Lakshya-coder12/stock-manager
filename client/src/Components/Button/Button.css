.btn {
  border: none;
  cursor: pointer;
  border-radius: 5px;
  background-color: #1976c2;
  color: white;
  padding: 8px 12px;
}
