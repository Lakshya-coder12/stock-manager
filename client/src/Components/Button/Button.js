import React from "react";
import "./Button.css";

function Button({ children, onClick, ...rest }) {
  return (
    <button {...rest} onClick={onClick} className="btn">
      {children}
    </button>
  );
}

export default Button;
