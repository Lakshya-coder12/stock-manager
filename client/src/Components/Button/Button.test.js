import { configure, shallow } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import Button from "./Button";

configure({ adapter: new Adapter() });

describe("Testing Button Component", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<Button />);
    expect(wrapper).toMatchSnapshot();
  });
});
