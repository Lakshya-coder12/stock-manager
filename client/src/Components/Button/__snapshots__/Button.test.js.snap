// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Button Component matches snapshot 1`] = `
<button
  className="btn"
/>
`;
